core/dbt/contracts/graph/unparsed.py

```diff
@@ -207,9 +207,16 @@ class UnparsedNodeUpdate(HasConfig, HasColumnTests, HasColumnAndTestProps, HasYa
     access: Optional[str] = None
 
 
+@dataclass
+class UnparsedCustomGranularity(dbtClassMixin):
+    name: str
+    column_name: Optional[str] = None
+
+
```

Comment:
Instead of redefining this class, can we instead just reuse the `CustomGranularity` class defined in core/dbt/artifacts/resources/v1/model.py. The classes are exactly the same _except_ in their class name (unless I'm misreading something, which is absolutely possible). We could actually do the same with the `UnparsedTimeSpine` vs `TimeSpine` class, but that is a separate matter.

Reply:
Good call! Removed duplicates for both!